tests/test_client.py

```diff
@@ -173,6 +177,25 @@ def create(self, stream, ext_obj=None, start_date=None):
             return [self.create_refunds(ext_obj, start_date).body.get('refund')]
         elif stream == 'payments':
             return [self.create_payments()]
+        elif stream == 'shifts':
+            employee_id = [employee['id'] for employee in self.get_all('employees')][0]
+            location_id = [location['id'] for location in self.get_all('locations')][0]
+
+            if not end_date:
```

Comment:
I think we should make the parameter shift_minutes with it defaulting to self.SHIFT_MINUTES (might need to pull the constant out of the class to refer to it as a default value). It's unclear to the person looking at the method signature right now what the default behavior is.

Reply:
We will address this later